app/src/main/java/org/simple/clinic/patient/PatientSaveModel.kt

```diff
@@ -0,0 +1,3 @@
+package org.simple.clinic.patient
+
+data class PatientSaveModel(val patient: Patient, val address: PatientAddress, val phoneNumbers: List<PatientPhoneNumber>)
```

Comment:
I try to avoid using suffixes when naming data classes whenever possible. It feels like one of those cases where people are accustomed to suffixing classes with verbs. For instance, we have `PatientSearchResult` instead of `PatientSearchResultModel` and so on.

At the same time, there are classes that use a suffix. For e.g., `SummaryAddNewBpListItem`, where we couldn't think of a better name. 

Do you think we can name `PatientSaveModel` to something better?

Reply:
Should we just call this `PatientProfile` then?

Reply:
Renamed to `PatientProfile`.